crates/core/migrations/2022-02-12-065030_add_not_null_contraint_to_token_account_amount/down.sql

```diff
@@ -0,0 +1 @@
+alter table token_accounts alter column amount drop not null;
```

Comment:
```suggestion
alter table token_accounts
alter column amount drop not null;
```

Reply:
well the migration won't run if I format like this :D

Reply:
@imabdulbasit that's weird, i've done it like that before in the other migrations

Reply:
Ok it works now after I recreated the db